CHANGELOG.md

```diff
@@ -1,5 +1,30 @@
 # Changelog
 
+## [0.7.2] - 2023-02-17
+
+### Features
+- Support the new SQL clause `WITH` (#2846 @aceforeverd)
```

Comment:
The with clause feature should mark experimental.

Reply:
> The with clause feature should mark experimental.

let's mark as [Alpha], please also do this in the doc